OpenSim/OpenSimDoxygenMain.h

```diff
@@ -30,10 +30,30 @@ OpenSim source and it is not installed with OpenSim. **/
 
 /** @mainpage  OpenSim 3.2 Documentation
 
+\if user
+This documentation is for people who want to use the OpenSim application
+programming interface (API) with the MATLAB, Python, Java or C++ languages.
+Therefore, internal (public/private) methods are hidden from this
+documentation. If you want to extend the OpenSim library by writing your own
+C++ plugin or you want to contribute directly to the OpenSim library, see the
+\a developer API documentation.
```

Comment:
Now that we support Markdown in doxygen, you can just write \`developer\` in back quotes which I think is a much nicer syntax that doxygen's `\a developer`. It also works for multiple word phrases.